### DISCUSSION TITLE
Add new color preset to toolbar?

### DISCUSSION THREAD

**Discussion by TunaSorbet** [xournalpp/xournalpp#3917] [Q&A] [answered]
Is there a way I can add a new color preset button to the toolbar? For example, purple is not one of the colors available on the toolbar. I know I can get purple by going into the select color menu, but for convenience's sake, I'd like to add it to my toolbar.

**Comment by rolandlo** [comment, marked as answer]
Yes, this is possible. Please consult the [website](https://xournalpp.github.io/guide/config/toolbar-colors/).

**Reply by TunaSorbet** [reply]
I was able to get it added to my toolbar. Thanks so much!
